### PR TITLE
AST-1577 - Custom font is not loading in Astra and Elementor customizer due to latest Elementor 5.6.0 update

### PR DESCRIPTION
**Pull request by GauravKhupse**
- The custom font is not showing in the Astra and Elementor customizer setting with the latest Elementor 3.6.0 release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,9 @@ If you're not using any of the supported plugins and theme, you can write the cu
 
 ## Changelog ##
 
+### 1.3.4 ###
+- Fix: Fixed custom font is not loading on Astra and Elementor customizer due to conflict with Elementor Pro.
```
Comment (premanshup) [inline README.md:47]:
```suggestion
- Fix: Custom fonts are not loading on Astra customizer and Elementor typography settings after Elementor Pro v3.6.0.
```

Comment (GauravKhupse) [reply README.md]: Changes done!

**Review by premanshup — APPROVED** [review]

File: includes/class-bsf-custom-fonts-taxonomy.php [hunk 2]
```diff
@@ -77,7 +77,7 @@ public function __construct() {
 		 *
 		 * @since 1.0.0
 		 */
-		public function create_custom_fonts_taxonomies() {
+		public static function create_custom_fonts_taxonomies() {
```
Comment (premanshup) [inline includes/class-bsf-custom-fonts-taxonomy.php:80]: @GauravKhupse We have called this function on object reference which should be converted to static call.
`add_action( 'init', array( $this, 'create_custom_fonts_taxonomies' ) );`

Comment (GauravKhupse) [reply includes/class-bsf-custom-fonts-taxonomy.php:80]: Changes done!

**Pull request merged**